example/package.json

```
@@ -9,7 +9,7 @@
     "moment": "^2.24.0",
     "react": "16.9.0",
     "react-native": "0.61.5",
-    "react-native-card-connect": "git+https://github.com/brij-dev/react-native-card-connect.git"
+    "react-native-card-connect": "git+https://github.com/BigRoomStudios/react-native-card-connect.git#install-gson-library"
```

```suggestion
    "react-native-card-connect": "git+https://github.com/BigRoomStudios/react-native-card-connect.git"
```

Commit this suggestion before merging in